packages/mdc-switch/README.md

```diff
@@ -70,17 +74,66 @@ Users can add the `disabled` attribute directly to the `<input>` element or a pa
 CSS Class | Description
 --- | ---
 `mdc-switch` | Mandatory, for the parent element.
-`mdc-switch__native-control` | Mandatory, for the input checkbox.
-`mdc-switch__background` | Mandatory, for the background element.
-`mdc-switch__knob` | Mandatory, for the knob element.
+`mdc-switch__track` | Mandatory, for the track element.
+`mdc-switch__thumb-underlay` | Mandatory, for the ripple effect.
+`mdc-switch__thumb` | Mandatory, for the thumb element.
+`mdc-switch__native-control` | Mandatory, for the hidden input checkbox.
+
+## Style Customization
+
+MDC Switch uses [MDC Theme](../mdc-theme)'s `secondary` color by default for the checked (toggled on) state. Use the following mixins to customize it.
 
 ### Sass Mixins
 
-The following mixins apply only to _enabled_ switches in the _on_ (checked) state.
-It is not currently possible to customize the color of a _disabled_ or _off_ (unchecked) switch.
+The following mixins apply only to _enabled_ switches.
+It is not currently possible to customize the color of a _disabled_ switch. Disabled switches use the same colors as enabled switches, but with a different opacity value.
 
 Mixin | Description
 --- | ---
-`mdc-switch-track-color($color)` | Sets the track color.
-`mdc-switch-knob-color($color)` | Sets the knob color.
-`mdc-switch-focus-indicator-color($color)` | Sets the focus indicator color.
+`mdc-switch-toggled-on-color($color)` | Sets the base color of the track, thumb, and ripple when the switch is toggled on.
+`mdc-switch-toggled-off-color($color)` | Sets the base color of the track, thumb, and ripple when the switch is toggled off.
+`mdc-switch-toggled-on-track-color($color)` | Sets color of the track when the switch is toggled on.
+`mdc-switch-toggled-off-track-color($color)` | Sets color of the track when the switch is toggled off.
+`mdc-switch-toggled-on-thumb-color($color)` | Sets color of the thumb when the switch is toggled on.
+`mdc-switch-toggled-off-thumb-color($color)` | Sets color of the thumb when the switch is toggled off.
+`mdc-switch-toggled-on-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled on.
+`mdc-switch-toggled-off-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled off.
+
+## `MDCSwitch` Properties and Methods
+
+Property | Value Type | Description
+--- | --- | ---
+`checked` | Boolean | Setter/getter for the switch's checked state
+`disabled` | Boolean | Setter/getter for the switch's disabled state
+
+## Usage within Web Frameworks
+
+If you are using a JavaScript framework, such as React or Angular, you can create a switch for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
```

Comment:
Capitalize "Switch" in "create a switch for your framework"

Reply:
Will do.